### PR TITLE
Km451 remove handlers and rest clients

### PR DESCRIPTION
**Pull request by ivaruf**

### CODE REVIEW

File: cmd/okctl/main.go [hunk 1]
```diff
@@ -168,8 +167,6 @@ func buildRootCommand() (*cobra.Command, *okctl.Okctl) {
 			o.Out = cmd.OutOrStdout()
 			o.Err = cmd.OutOrStderr()
 
-			o.SetFormat(core.EncodeResponseType(outputFormat))
```
Comment (deifyed) [inline cmd/okctl/main.go:171]: This doesn't have anything to do with client/server? I don't think we're using it though

File: pkg/okctl/okctl.go [hunk 2]
```diff
@@ -576,11 +543,6 @@ func (o *Okctl) CoreServices() (core.Services, error) {
 		run.NewKubeRun(o.CloudProvider, o.CredentialsProvider.Aws()),
 	)
 
-	awsIamAuth, err := o.BinariesProvider.AwsIamAuthenticator(awsiamauthenticator.Version)
-	if err != nil {
-		return core.Services{}, err
-	}
```
Comment (deifyed) [inline pkg/okctl/okctl.go:582]: Why remove this?

**Review by deifyed — COMMENTED** [review]
I'll approve as soon as you reply to the question about removing the auth thing

File: pkg/okctl/okctl.go (hunk 2 above)
Comment (fredriv) [reply pkg/okctl/okctl.go:582]: It's moved up to line 487?

Comment (deifyed) [reply pkg/okctl/okctl.go:582]: My bad. Shouldn't read big PRs before 0900

**Review by deifyed — APPROVED** [review]
Nice!! A major improvement

File: cmd/okctl/main.go (hunk 1 above)
Comment (ivaruf) [reply cmd/okctl/main.go:171]: Format for http requests - that is going away :)

**Pull request merged**